### DISCUSSION TITLE
Fourier series components

### DISCUSSION THREAD

**Discussion by GrzegorzRut** [alan-turing-institute/sktime#2649] [Ideas]
Hi!
I've recently stumbled upon [this](https://robjhyndman.com/hyndsight/longseasonality/) and [this](https://medium.com/intive-developers/forecasting-time-series-with-multiple-seasonalities-using-tbats-in-python-398a00ac0e8a).

As far as I can tell, `sktime` so far does not have functions like [fourier](https://www.rdocumentation.org/packages/forecast/versions/8.16/topics/fourier). We do have the `CosineTransformer` but this can probably be extended to something like the `fourier` function mentioned above. Seems to be really helpful with for long or multiple seasonalities.

Would it be helpful to have something like that?

**Comment by fkiraly** [comment]
Hm, the legacy estimator overview has a `FourierTransformer` in it, but indeed this does not seem to exist in the repository.

Good idea!

**Reply by fkiraly** [reply]
converted into a good first issue: https://github.com/alan-turing-institute/sktime/issues/2650
